src/components/TodoApp/TodoApp.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import uuid from 'uuid/v4';
+import PropTypes from 'prop-types';
+
+export class TodoApp extends React.Component {
+  state = {
+    title: '',
+  }
+
+  handleChange = (event) => {
+    this.setState({
+      title: event.target.value,
+    });
+  }
+
+  handleKeyPress = (event) => {
+    const { title } = this.state;
+
+    if (event.key === 'Enter') {
```

Comment:
add form so that you don't need to find key name

Reply:
didn't get, what form and where should I add it?